pkgs/applications/networking/cluster/nomad/default.nix

```diff
@@ -38,9 +38,10 @@ let
       '';
 
       meta = with lib; {
+        inherit knownVulnerabilities;
         homepage = "https://www.nomadproject.io/";
         description = "A Distributed, Highly Available, Datacenter-Aware Scheduler";
-        license = licenses.mpl20;
+        license = licenses.bsl11;
```

Comment:
1.4 is still mpl 2.0

Reply:
Ah damn I was hoping to avoid the bother of adding a passthrough mechanism.